src/core/00_github_runner.tf

```diff
@@ -30,3 +30,96 @@ module "github_runner" {
 
   tags = var.tags
 }
+
+locals {
+  repo_owner = "PagoPA"
+  repo_name  = "io-infra"
+  image_name = "ghcr.io/pagopa/github-self-hosted-runner-azure:beta-runner-v2"
+}
+
+data "azurerm_key_vault" "keyvault" {
+  name                = "io-p-kv-common"
+  resource_group_name = "io-p-rg-common"
+}
```

Comment:
I'm quite sure that there is another data or module in src folder that reference the same kv.
I think it's better to reference that kv